packages/vite-plugin-cloudflare/playground/dev-vars/.dev.vars.staging

```diff
@@ -0,0 +1,3 @@
+ENV_NAME = "staging"
+MY_DEV_VAR_A = "my .dev.vars staging variable A"
+MY_DEV_VAR_B = "my .dev.vars staging variable B"
```

Comment:
```suggestion
MY_DEV_VAR_A = "my .dev.vars.staging variable A"
MY_DEV_VAR_B = "my .dev.vars.staging variable B"
```

Reply:
I prefer not to have the `.` so that it is clearer that this this is a manually inputted string (otherwise without seeing the implementation someone could thing that this was generated by something like `my ${devVarsFileName} variable A`)

if you find it confusing I can change the strings to something like `MY_DEV_VAR_A = "my .dev.vars variable A for the staging environment` (or something like that)